andela/travel_tool_back
# #159236231 Setup Sequlize

#### What does this PR do?
Setup database ORM and migrations

#### Description of Task to be completed?
- Set up the migrations and models' structure
- Create a dummy user table and a model setup to test the configuration
- Add setup guide to project README

#### How should this be manually tested?
This can be tested by cloning the repo changing into the root project directory then the following sequence of steps:
- Install the project dependencies with `yarn install`
- See [this](https://github.com/andela/travel_tool_back/tree/ch-setup-sequelize-orm-159236231#database-and-orm) README guide on how to set up the database locally
- Run the server with `yarn run start:dev`
- Send a POST Request to `http://localhost:5000/api/v1/register`. Include in the body `json` data with `firstName`, `lastName` properties and an `email` property with domain `andela.com`
- If everything is okay, you should get a success response

#### What are the relevant pivotal tracker stories?
#159236231

#### Screenshots
##### The Request
<img width="812" alt="screen shot 2018-07-26 at 10 43 38 am" src="https://user-images.githubusercontent.com/23254512/43248634-27e4f7ec-90c1-11e8-9174-a42e1f9e291c.png">

##### The Response
<img width="774" alt="screen shot 2018-07-26 at 10 46 56 am" src="https://user-images.githubusercontent.com/23254512/43248694-5b2bd4e0-90c1-11e8-825f-95a5b981c70f.png">
